share/management/commands/loadsources.py

```diff
@@ -23,8 +21,14 @@ def add_arguments(self, parser):
         parser.add_argument('--overwrite', action='store_true', help='Overwrite existing sources and source configs')
 
     def handle(self, *args, **options):
+        global apps
```

Comment:
It's probably a better idea to just tack this onto `self`. Not sure why I didn't think of that first.